t/01-wasm/002-shm_queue_sanity.t

```diff
@@ -3,6 +3,10 @@
 use strict;
 use lib '.';
 use t::TestWasm;
+use POSIX qw(sysconf _SC_PAGESIZE);
+
+our $page_size = sysconf(_SC_PAGESIZE);
```

Comment:
`TestWasm.pm` seems like a good place for `$page_size` declaration and definition, although only `shm` tests will potentially benefit from it.

Reply:
Yes, moving it to `TestWasm.pm` makes sense to me, even `min_shm_size` if it makes sense to you.